### DISCUSSION TITLE
Drop support for 32bit architectures: armeabi-v7a and x86?

### DISCUSSION THREAD

**Discussion by tagcode** [stride3d/stride#1599] [General]
Should support for 32bit targets be dropped in favor for simpler build maintenance?

64bit [x86-64](https://en.wikipedia.org/wiki/X86-64) emerged 24 years ago and [arm64-v8a](https://en.wikipedia.org/wiki/AArch64) 12. How much 32bit hardware can there still be? Hardware that this codebase compiles against to with available toolchains?

**Comment by tagcode** [comment]
I cannot get 32bit Android target to build with the NDK tools that get installed with latest Android Studio. If I remove "armeabi-v7a" and "x86" from Stride.Native.targets, building works. Is there any reason to keep these supported? [See](https://github.com/stride3d/stride/discussions/1601).

**Reply by manio143** [reply]
It's hard to say if there's anyone using 32bit Stride builds - I would guess probably not. If they cause problems building and the market share of operating systems that require those platforms is not existent then we should remove them. Unless somebody whose interested in building for them comes along to help support it.

**Reply by NicusorN5** [reply]
x86 systems should be considered ancient. I support removing x86 support for Stride.

**Reply by tagcode** [reply]
32bit arm exists only in nano scale embedded systems that will not run stride due to their constrains on memory, performance, gpu and purpose.

**Reply by tagcode** [reply]
Raspberry Pi has had 64bit hardware since Pi 3 (2016), so 7 years ago.
